start.py

```diff
@@ -285,11 +285,14 @@ def cm_server_not_dead(primary_node):
 
     if args.kafka_version:
         logger.info('Configuring Kafka ...')
-        _configure_kafka(deployment, cluster, kafka_version=args.kafka_version)
+        _configure_kafka(deployment, cluster, kafka_version=args.kafka_version, cdh_version=args.cdh_version)
 
-    if args.kudu_version:
+    if Version(args.cdh_version) >= Version('5.13.0'):
```

Comment:
This doesn't seem like the right logic since it seems like it'll configure Kudu whether you want it or not as long as CDH version is greater than 5.13. Instead, we only want to configure Kudu if users ask for it (since it's not part of CDH). We just need to skip the parcel stuff if the CDH version is 5.13+.

Reply:
Done.